modules/core/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-gcp/src/main/java/org/nuxeo/ecm/core/storage/gcp/GoogleStorageBinaryManager.java

```diff
@@ -186,8 +203,14 @@ public void storeFile(String digest, File file) {
             String key = bucketPrefix + digest;
             // try to get the blob's metadata to check if it exists
             if (bucket.get(key) == null) {
-                try (var fis = new FileInputStream(file)) {
-                    bucket.create(key, fis);
+                int bufferLength;
```

Comment:
`bufferLength` definition can be moved inside the `try`.
Probably `buffer` too.